samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -229,16 +229,12 @@ LPN configuration
 =================
 
 To make the light switch run as an LPN, set :makevar:`EXTRA_CONF_FILE` to :file:`overlay-lpn.conf` when building the sample.
-For example, when building from the command line, use the following command, where *board_target* is the target for the development kit for which you are building:
-
-.. parsed-literal::
-   :class: highlight
-
-   west build -b *board_target* -p -- -DEXTRA_CONF_FILE="overlay-lpn.conf"
```

Comment:
It will nice to retain this at least as one direct example. After this text you could say:

```
See :ref:`cmake_options` for more information on how to use additional build variables.
```

Reply:
I ended up looking for this exact command today in the docs. It is useful I guess. Thanks.